src/App.tsx

```diff
@@ -1,16 +1,26 @@
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
+import { Movie } from './types/Movie';
 import moviesFromServer from './api/movies.json';
+import { useState } from 'react';
+
+const initialMovies: Movie[] = moviesFromServer;
 
 export const App = () => {
+  const [movies, setMovies] = useState<Movie[]>(initialMovies);
```

Comment:
```suggestion
export const App = () => {
  const [movies, setMovies] = useState<Movie[]>(moviesFromServer);
```
You can pass `moviesFromServer` into state directly